packages/stable/src/__tests__/testUtils.ts

```diff
@@ -186,6 +183,94 @@ const getFileCreateArgs = (
 
   return { postfix, fileContent, sourceCreatedTime, localFileMeta };
 };
+export function setupMockableClientForIntegrationTests() {
+  const client = setupClient(process.env.COGNITE_BASE_URL);
+  return client;
+}
+export function setupLoggedInClientForUnitTest() {
+  return new CogniteClient({
+    appId: 'JS SDK integration tests (beta)',
```

Comment:
The appId should perhaps not include the "(beta)" keyword in the name now, as it is moved to stable?